virttest/qemu_vm.py

```diff
@@ -4061,6 +4071,7 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
             os.close(fd_src)
 
         clone = self.clone()
+        clone.images_offline_snapshots = self.devices.images_offline_snapshots
```

Comment:
Checked that qemu would do `unlink` to the images, which means destination ends would never touch them, so we don't have to support it with migration, let's make it work at its local end.